codeserver/che-code-ubi9-python-3.12/run-nginx.sh

```
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+source /opt/app-root/etc/generate_container_user
+
+set -e
+
+source ${NGINX_CONTAINER_SCRIPTS_PATH}/common.sh
```

Shellcheck (0.11.0)

[info] Not following: ./common.sh was not specified as input (see shellcheck -x).

(SC1091)